ambry-tools/src/main/java/com.github.ambry/tools/admin/DirectoryUploader.java

```diff
@@ -146,7 +148,8 @@ public void walkDirectoryToCreateBlobs(String path, FileWriter writer, String da
           throw new IllegalArgumentException("File length is " + f.length() + "; files larger than " + Integer.MAX_VALUE
               + " cannot be put using this tool.");
         }
-        BlobProperties props = new BlobProperties(f.length(), "migration");
+        BlobProperties props =
+            new BlobProperties(f.length(), "migration", Account.UNKNOWN_ACCOUNT_ID, Container.UNKNOWN_CONTAINER_ID);
```

Comment:
Would it be a problem if we explicitly put to `UNKNOWN_ACCOUNT` and `UNKNOWN_CONTAINER`?